integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -2,7 +2,12 @@
 FROM ubuntu:22.04 as chip-build-cert
 ARG TARGETPLATFORM
 # COMMITHASH defines the target commit to build from. May be passed in using --build-arg.
-ARG COMMITHASH=1e06479f412b78eea0dfbeacf7cad51d005b649b
+ARG COMMITHASH=451e602649b8b37ba6c9aab8ba5e30747293f778
+
+# ZAP Development install, so that it runs on both x64 and arm64
+# Generally this should match with the ZAP version that is used for codegen within the
+# specified SHA
+ARG ZAP_VERSION=v2023.01.05-nightly
```

Comment:
https://github.com/project-chip/connectedhomeip/pull/24306 seems to move to v2023.01.06-nightly, so should this also use that as well?

Reply:
We may need to do this at some point ... generally these should be in sync. Thinking separate PR though that should be done when the new image starts being used in CI (i.e. when the workflow validating zap is changed).